src/components/shared/switcherTheme/SwitcherTheme.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC } from 'react';
+
+import { useMantineColorScheme, Switch, useMantineTheme } from '@mantine/core';
+import { LuSun } from 'react-icons/lu';
+import { PiMoonStars } from 'react-icons/pi';
+
+import {
+  iconSize,
+  iconColorMoon,
+  iconColorSun,
+} from 'components/shared/switcherTheme/config';
+import { ColorScheme } from 'components/shared/switcherTheme/types';
+
+const SwitcherTheme: FC = () => {
+  const { colorScheme, toggleColorScheme } = useMantineColorScheme();
+  const theme = useMantineTheme();
+
+  const sunIcon = <LuSun style={iconSize} color={theme.colors.yellow[iconColorSun]} />;
+
+  const moonIcon = (
+    <PiMoonStars style={iconSize} color={theme.colors.blue[iconColorMoon]} />
+  );
```

Comment:
what is `()` for?

Reply:
These brackets are added by the linter because the string is long and it performs the hyphenation.